bddl/activity_definitions/cleaning_barbecue_grill/problem0.bddl

```diff
@@ -8,20 +8,20 @@
     	floor.n.01_1 - floor.n.01
     	rag.n.01_1 - rag.n.01
     	bucket.n.01_1 - bucket.n.01
-        table.n.02_1 - table.n.02
+        countertop.n.02_1 - countertop.n.02
```

Comment:
should be `countertop.n.01` , not `countertop.n.02`. 

Make sure to run the BDDL tests (from `bddl/tests`, run `python bddl_tests.py batch_verify`). This will catch errors like invalid synsets, etc.

Reply:
Fixed. Thanks for the tip